test_util/run-all

```diff
@@ -33,7 +33,7 @@ else
 fi
 
 # Pin to just the right dcos-vagrant commit
-git -C dcos-vagrant checkout -qf 2b323c4c1942e3f6d484a7375e749929356a4417
+git -C dcos-vagrant checkout -qf 03de5522ba6da57b3f624fcd9985545f70a89497
```

Comment:
It's unclear why this change and the one below are necessary. Can you add something to the PR description that explains this?